custom_components/foxess_modbus/entities/xx1_aux_entity_descriptions.py

```diff
@@ -167,6 +184,8 @@
         state_class=SensorStateClass.TOTAL,
         native_unit_of_measurement="kWh",
         scale=0.1,
+        # currently overflows
```

Comment:
I'd expect this might be negative if the inverter took more energy from the AC side than it output?

Reply:
Unsure, have left it commented out for now.